css-flexbox-photo-gallery/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
```

Comment:
You could remove `<meta http-equiv="X-UA-Compatible" content="ie=edge">`. Since this project falls after the "Accessibility User Feedback Survey" project, it would be good review to write a simple `alt` description for each photo. Finally, this could be renamed to `index-final.html`.

Reply:
Learners will build this project line by line, starting with the HTML. You could start the project here with no code and just a commented out section similar to the following:

```
<!-- 
    Begin by creating the structure of the page with the doctype and four main HTML elements:

    ```
    <!DOCTYPE html>
    <html>
    <head>
        <title></title>
    </head>
    <body>
    </body>
    </html>
    ```
-->
```

The next steps could be adding the title, meta elements, linking to the style sheets, and adding the header to the body of the page. The HTML after these 5 to 10 steps could look like this:

```
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>CSS Flex Photo Gallery</title>
    <link rel="stylesheet" href="./css/style50.css">
</head>
<body>
    <!-- Header -->
    <div class="header">
        <h1>CSS FLEXBOX PHOTO GALLERY</h1>
    </div>
</body>
</html>
```